src/components/pages/error/Error.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { useNavigate } from "react-router-dom"
+
+export default function Error() {
+
+  const Navigate = useNavigate();
```

Comment:
```suggestion
  const navigate = useNavigate();
```

useNavigate() ne retourne pas un composant mais une fonction. Donc pas de majuscule à Navigate